internal/cephfs/core/volume.go

```diff
@@ -213,6 +215,7 @@ func (s *subVolumeClient) CreateVolume(ctx context.Context) error {
 	// and if not, initialize with default values(false).
 	if _, keyPresent := clusterAdditionalInfo[s.clusterID]; !keyPresent {
 		clusterAdditionalInfo[s.clusterID] = &localClusterState{}
+		clusterAdditionalInfo[s.clusterID].subVolumeGroupsCreated = make(map[string]bool)
```

Comment:
add this in the following lines too ?
https://github.com/ceph/ceph-csi/blob/9d46478794721f2887904ebe6340ef03c5a8aa45/internal/cephfs/core/metadata.go#L37
https://github.com/ceph/ceph-csi/blob/9d46478794721f2887904ebe6340ef03c5a8aa45/internal/cephfs/core/snapshot_metadata.go#L33